src/index.html

```diff
@@ -11,6 +11,39 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <input
+    data-qa="big"
+    class="search-bar-big"
+    type="search"
+    placeholder="Try &#34Los Angeles&#34"
+    >
```

Comment:
[checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) 5th item, fix your attribute indentation everywhere, not just here
```suggestion
    <input
      data-qa="big"
      class="search-bar-big"
      type="search"
      placeholder="Try &#34Los Angeles&#34"
    >
```